gobblin-service/src/main/java/org/apache/gobblin/service/modules/core/GobblinServiceGuiceModule.java

```diff
@@ -194,6 +195,8 @@ public void configure(Binder binder) {
       binder.bind(Orchestrator.class);
       binder.bind(SchedulerService.class);
       binder.bind(GobblinServiceJobScheduler.class);
+      OptionalBinder.newOptionalBinder(binder, UserQuotaManager.class);
```

Comment:
If this is needed?

Reply:
I think it's needed since the quota manager is tied directly with the dagmanager. Since the dagmanager is optional I made the quota manager optional. And I wasn't able to get the code to run without the above binding done.